From 94a778c9aa561c8cbe5ef39e21695db3124611b4 Mon Sep 17 00:00:00 2001
Message-Id: <94a778c9aa561c8cbe5ef39e21695db3124611b4.1566258525.git.liu.denton@gmail.com>
In-Reply-To: <cover.1566258525.git.liu.denton@gmail.com>
References: <cover.1566258525.git.liu.denton@gmail.com>
From: Denton Liu <liu.denton@gmail.com>
Date: Mon, 19 Aug 2019 19:24:37 -0400
Subject: [PATCH v2 3/4] config/format.txt: make clear the default value of
 format.coverLetter
To: Git Mailing List <git@vger.kernel.org>
Cc: Ævar Arnfjörð Bjarmason <avarab@gmail.com>,
    Junio C Hamano <gitster@pobox.com>,
    Eric Sunshine <sunshine@sunshineco.com>

Signed-off-by: Denton Liu <liu.denton@gmail.com>
---
 Documentation/config/format.txt | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/Documentation/config/format.txt b/Documentation/config/format.txt
index 414a5a8a9d..60e4e92885 100644
--- a/Documentation/config/format.txt
+++ b/Documentation/config/format.txt
@@ -74,7 +74,8 @@ format.signOff::
 	Please see the 'SubmittingPatches' document for further discussion.
 
 format.coverLetter::
-	A boolean that controls whether to generate a cover-letter when
+	A boolean which lets you enable the `--cover-letter` option by
+	default when
 	format-patch is invoked, but in addition can be set to "auto", to
 	generate a cover-letter only when there's more than one patch.
 
-- 
2.23.0.248.g3a9dd8fb08

